app/templates/components/hyper-table/column.hbs

```diff
@@ -1,9 +1,15 @@
 <header>
-  <div class="cell-header" data-toggle="tooltip" data-title={{column.field.name}} data-placement="bottom" {{action "orderColumn"}}>
-    {{#if column.field.title}}
-      <span class="text-style-capitalize">{{column.field.title}}</span>
-    {{else}}
-      {{column.field.name}}
+  <div class="cell-header" {{action "orderColumn"}}>
+    <span data-toggle="tooltip" data-title={{column.field.name}} data-placement="bottom">
+      {{#if column.field.title}}
+        <span class="text-style-capitalize">{{column.field.title}}</span>
+      {{else}}
+        {{column.field.name}}
+      {{/if}}
+    </span>
+
+    {{#if column.field.tooltip}}
+      <i data-toggle="tooltip" data-title={{column.field.tooltip}} data-html="true" data-placement="bottom" class="fa fa-info-circle header-tooltip"></i>
```

Comment:
if the `column.field.tooltip` attr is html, you might need to use `{{{ }}}` instead, no ?

Reply:
Hmm good point, but it's working correctly without the `{{{ }}}` I think because of `data-html`, the tooltip library probably handles the parsing